### ISSUE TITLE
Automatic comparison of plots

### ISSUE THREAD

**Issue by tassos** [tassos/matlab_tutorial#3]
Some students had the idea to compare the plots automatically. We could basically convert the plots into two image grids and then substract them. If result is all zeros, it means the plots are identical.

**Comment by Zoltan21** [comment]
yep, good idea :) have you finished with the class? Can I call you?
    On Wednesday, October 3, 2018, 3:20:00 PM GMT+3, Tassos Natsakis <notifications@github.com> wrote:  
 
 
Some students had the idea to compare the plots automatically. We could basically convert the plots into two image grids and then substract them. If result is all zeros, it means the plots are identical.
